### PR TITLE
RHTAP-3032 Rework env vars to use values from RHTAP installation in case

### PR DESCRIPTION
**Pull request by jkopriva**
default.env is empty.

### CODE REVIEW

File: src/apis/ci/jenkins.ts [hunk 1]
```diff
@@ -3,26 +3,21 @@ import { Utils } from '../git-providers/utils';
 
 export class JenkinsCI extends Utils {
     // Jenkins server details
-    private JENKINS_URL: string;
-    private JENKINS_USER: string;
-    private JENKINS_API_TOKEN: string;
-    private axiosInstance: Axios;
+    private readonly JENKINS_URL: string;
```
Comment (flacatus) [inline src/apis/ci/jenkins.ts:6]: why uppercase?

Comment (jkopriva) [reply src/apis/ci/jenkins.ts]: Renamed - thanks! Resolving.

**Pull request merged**